src/elements/Header/Subheader.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+
+export default class Subheader extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+  static _meta = {
+    library: META.library.semanticUI,
+    name: 'Sub',
+    type: META.type.element,
+  };
+  render() {
+    const classes = classNames(
+      'sd-sub',
+      'sub',
+      this.props.className,
+      'header',
+    );
+    return (
+      <div className={classes}>
+        {this.props.children}
+      </div>
+    );
+  }
+}
```

Comment:
Component for the `.Subheader` addition. Returns a `<div class="sub header"> and can take additional classNames, although I'm not sure if Semantic has any others for the subheader